GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -132,6 +132,11 @@ void GPUTRDTracker_t<TRDTRK, PROP>::InitializeProcessor()
     Error("Init", "TRD geometry must be provided externally");
   }
 
+  SetPtThreshold(Param().rec.trdMinTrackPt);
```

Comment:
If I understand correctly here you copy the properties to inside the TRDTracker. Is there any particular reason to do so? Why don't you just access it directly where it is used?

Reply:
Hm.. If I would use the parameter directly would there be a way to change it in AliRoot without having to recompile?

Reply:
I mean if that is the case it is probably not too bad, since the development is done and I will not need that functionality so frequently anymore

Reply:
Well, how do you change it in AliRoot right now?
Don't you have a ROOT macro where you pass in the all the config objects? You can just change it there before you call the SetSettings? Shouldn't work in interpreted mode, since there is no dictionary, but in compiled mode it should be ok.

Reply:
I initialized the GPUSettingsRec struct, used the default values and afterwards changed the settings with setter functions of the GPUTRDTracker... I see now that this can be done in a more elegant way :)